optuna/visualization/matplotlib/_contour.py

```diff
@@ -175,7 +175,9 @@ def _calculate_axis_data(
     returned_values: Sequence[int | float]
     if axis.is_cat:
         enc = _LabelEncoder()
-        returned_values = enc.fit_transform(list(map(str, values)))
+        # prepare all values in categorical distribution
```

Comment:
A nit, but please follow the coding convention: "Block and inline comments are supposed to be sentence(s)."
https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentences

```suggestion
        # Prepare all values in categorical distributions.
```